src/test/java/org/gridsuite/modification/server/ModificationRepositoryTest.java

```diff
@@ -255,6 +255,55 @@ public void testGeneratorCreation() {
         );
     }
 
+    @Test
+    public void testShuntCompensatorCreation() {
+        var shunt1 = ShuntCompensatorCreationInfos.builder()
+            .equipmentId("shunt1").equipmentName("nameOne")
+            .currentNumberOfSections(1).maximumNumberOfSections(2)
+            .susceptancePerSection(1.).isIdenticalSection(true)
+            .voltageLevelId("vlId1").busOrBusbarSectionId("busId1")
+            .build();
+        var shunt2 = ShuntCompensatorCreationInfos.builder()
+            .equipmentId("shunt2").equipmentName("notNameOne")
+            .currentNumberOfSections(1).maximumNumberOfSections(2)
+            .susceptancePerSection(1.).isIdenticalSection(true)
+            .voltageLevelId("vlId1").busOrBusbarSectionId("busId1")
+            .build();
+
+        var createShuntCompensatorEntity1 = modificationRepository.createShuntCompensatorEntity(shunt1);
+        var createShuntCompensatorEntity2 = modificationRepository.createShuntCompensatorEntity(shunt2);
+
+        modificationRepository.saveModifications(TEST_GROUP_ID, List.of(createShuntCompensatorEntity1, createShuntCompensatorEntity2));
+        assertRequestsCount(1, 5, 0, 0);
+
+        List<ModificationInfos> modificationInfos = modificationRepository.getModifications(TEST_GROUP_ID, true);
+        assertEquals(2, modificationInfos.size());
+
+        assertThat(modificationRepository.getShuntCompensatorCreationModification(TEST_GROUP_ID, modificationInfos.get(0).getUuid()),
+            MatcherShuntCompensatorCreationInfos.createMatcher(createShuntCompensatorEntity1.toModificationInfos()));
+        assertThat(modificationRepository.getShuntCompensatorCreationModification(TEST_GROUP_ID, modificationInfos.get(1).getUuid()),
+            MatcherShuntCompensatorCreationInfos.createMatcher(createShuntCompensatorEntity2.toModificationInfos()));
+
+        assertEquals(2, modificationRepository.getModifications(TEST_GROUP_ID, true).size());
+        assertEquals(List.of(TEST_GROUP_ID), this.modificationRepository.getModificationGroupsUuids());
+
+        SQLStatementCountValidator.reset();
```

Comment:
is it pertinent to test modification deletion for each type of equipment ? 
It could be test once in a test on modification